transaction-decoder-tool/src/test/kotlin/linea/test/MainRunner.kt

```diff
@@ -0,0 +1,61 @@
+package linea.test
+
+import io.vertx.core.Vertx
+import net.consensys.linea.async.get
+import org.apache.logging.log4j.Level
+import org.apache.logging.log4j.LogManager
+import org.apache.logging.log4j.core.config.Configurator
+import java.util.concurrent.TimeUnit
+
+fun configureLoggers(loggerConfigs: List<Pair<String, Level>>) {
+  loggerConfigs.forEach { (loggerName, level) ->
+    Configurator.setLevel(loggerName, level)
+  }
+}
+
+fun main() {
+  val rplUrl = run {
+    "https://linea-sepolia.infura.io/v3/${System.getenv("INFURA_PROJECT_ID")}"
```

Comment:
Maybe it's easier to put the whole endpoint into an env var and make it generic?

Reply:
What would you can that env var?

Reply:
`RPC_URL` (typo in `rplUrl`, btw)

Reply:
I remember now why I didn't do env var, because to read from ENV we need to restart IntellIJ, which only loads ENV once on start, very inconvenient to bounce from environments. Which I did quite often so I find this more convenient for prototyping. Will fix the typo though :)